### PR TITLE
Has been added the Profile service (CRUD)

### PR DESCRIPTION
**Pull request by AndreiBujenitsa**

### CODE REVIEW

File: lib/cardconnect/services/authorization/authorization_response.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ class AuthorizationResponse
       include Utils
 
       FIELDS = [:respstat, :retref, :account, :token, :amount, :merchid, :respcode,
-                :resptext, :respproc, :avsresp, :cvvresp, :authcode, :commcard, :profileid]
+                :resptext, :respproc, :avsresp, :cvvresp, :authcode, :commcard, :profileid, :acctid]
```
Comment (thetizzo) [inline lib/cardconnect/services/authorization/authorization_response.rb]: Does this field come over when getting an Auth response from CardConnect's API?  It does not appear to be in their documentation unless I am missing something.

**Comment by thetizzo** [comment]
Thanks for the PR!  Would you be willing to add some tests and update the documentation for Profile service in the README?

File: lib/cardconnect/services/authorization/authorization_response.rb (hunk 1 above)
Comment (AndreiBujenitsa) [inline lib/cardconnect/services/authorization/authorization_response.rb]: Yes, look at https://developer.cardconnect.com/cardconnect-api#profiles

**Comment by AndreiBujenitsa** [comment]
about  tests and documentation... I'll do it

File: lib/cardconnect/services/authorization/authorization_response.rb (hunk 1 above)
Comment (thetizzo) [inline lib/cardconnect/services/authorization/authorization_response.rb]: Noted. LGTM

**Comment by thetizzo** [comment]
Thanks!

**Comment by thetizzo** [comment]
@AndreiBujenitsa Hey, I found some time to do the tests for this so I will merge it and handle that part.  Thanks for the contribution!

**Pull request merged**